rest/utilities_testing.go

```diff
@@ -66,7 +66,7 @@ func NewRestTester(tb testing.TB, restConfig *RestTesterConfig) *RestTester {
 	if restConfig != nil {
 		rt.RestTesterConfig = restConfig
 	} else {
-		rt.RestTesterConfig = &RestTesterConfig{}
+		rt.RestTesterConfig = &RestTesterConfig{noAdminParty: true}
```

Comment:
This approach doesn't disable GUEST by default for tests specifying a RestTesterConfig.

A preferred approach would be to change the parameter from "noAdminParty" to something like "guestEnabled", so that the default if not specified is false.

Reply:
I was thinking in the other way, I thought noAdminParty should be kept as is. Renaming "noAdminParty" to something like "guestEnabled" requires changes in lot of tests than we expected. Let me review all those tests and make this PR up, thank you!